www/tools/fetchComponentList.js

```diff
@@ -0,0 +1,82 @@
+import https from 'https';
+import fileSave from 'file-save';
+import path from 'path';
+import { kebabCase } from 'lodash';
+
+const GITHUB_CONTENT_URL = process.env.GITHUB_CONTENT_URL || 'https://raw.githubusercontent.com/momentum-design';
+
+const COMPONENT_LIST_URLS = {
+  'core':`${GITHUB_CONTENT_URL}/momentum-ui/master/core/data/components.json`,
+  'react':`${GITHUB_CONTENT_URL}/momentum-ui/master/react/src/app/ComponentList.js`,
+  'angular':`${GITHUB_CONTENT_URL}/momentum-ui/master/angular/src/lib/public_api.ts`,
+  'vue':`${GITHUB_CONTENT_URL}/momentum-ui/master/vue/components.json`
+};
+
+
+const PATH_JSON = path.resolve(__dirname, '../client/data');
+
+const save = (filePath, fileName, data) => {
+  fileSave(path.join(filePath, fileName))
+    .write(data, 'utf8')
+    .end('\n');
+  // eslint-disable-next-line no-console
+  console.log(`Component list saved to ${filePath}/${fileName}`);
+};
+
+const getReactComponentList = str => {
+  let res = str.match(/'(.+?)'/g);
+  return res.map(function(item){
+    return kebabCase(item.replace(/'/g, ''));
+  });
+};
+
+const getAngularComponentList = str => {
+  let res = str.match(/'\.\/(.+?)\//g);
+  return res.map(function(item){
+    return item.replace(/'\.\/|\//g, '');
+  });
+};
+
+const downloadFile = (fileName) => {
```

Comment:
This should be updated to get the file from the filesystem and not GitHub.com. Also, the name "download" will no longer make sense here.